src/hashbang.html

```diff
@@ -54,11 +54,11 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black"/>
     <meta name="HandheldFriendly" content="true"/>
     <meta name="MobileOptimized" content="320"/>
-    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
-    <link href='/assets/local.css' rel='stylesheet' type='text/css'>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700" rel="stylesheet" type="text/css"> 
```

Comment:
Why not use `font-weight: bold;` in local.css? 

I'm also not entirely certain this PR does what it says on the tin. I tried the relevant html locally and saw no cosmetic difference between the modified code and what's in production in either Chrome or Firefox. 

I would find defining what the look of the #! site should be extremely helpful.